pynance/data/retrieve.py

```diff
@@ -10,14 +10,14 @@
 Wraps `Pandas Remote Data Access 
 <http://pandas.pydata.org/pandas-docs/stable/remote_data.html>`_.
 """
-
+from datetime import datetime
 from ftplib import FTP
 from functools import partial
 import io
 
 import pandas_datareader.data as web
 
-def get(equity, start, end):
+def get(equity, start, end = datetime.today().date().isoformat()):
```

Comment:
The iso formatted date just gets changed back into a datetime object anyway:

- https://github.com/pydata/pandas-datareader/blob/823e6fc6b82c6d1cd5645bdc4da87094e7773f51/pandas_datareader/yahoo/daily.py#L60
- https://github.com/pydata/pandas-datareader/blob/823e6fc6b82c6d1cd5645bdc4da87094e7773f51/pandas_datareader/yahoo/daily.py#L60

I also noticed that the DataReader object allows end (and even start) to be None:

    >>> df = web.DataReader('amzn', 'yahoo', '2016-02-01')
    >>>

The best way might be just to let the DataReader handle it:

   if end == None:
       return web.DataReader(equity, 'yahoo', start)
   ...